app/styles/base.css

```diff
@@ -790,19 +790,23 @@ select:active {
  * ----------------------------------------
  */
 
-#noVNC_password_dlg {
+#noVNC_credentials_dlg {
   position: relative;
 
   transform: translateY(-50px);
 }
-#noVNC_password_dlg.noVNC_open {
+#noVNC_credentials_dlg.noVNC_open {
   transform: translateY(0);
 }
-#noVNC_password_dlg ul {
+#noVNC_credentials_dlg ul {
   list-style: none;
   margin: 0px;
   padding: 0px;
 }
+.noVNC_hidden {
```

Comment:
We already have `.noVNC_hidden` in some more specific places. Having it unqualified like this can get a bit confusing. Perhaps we should drop the others and have a generic one like this?

@samhed?

Reply:
Yep, I didn't quite got the point of having specific ones all over the place, all doing exactly the same thing, tbh

Reply:
> Perhaps we should drop the others and have a generic one like this?

sounds good to me